### ISSUE TITLE
PlatformInfo plugin returns zero values for SafeAreaInsets on Cordova Android

### ISSUE THREAD

**Issue by skymen** [Scirra/Construct-3-bugs#4331]
## Problem description

PlatformInfo plugin returns 0 values from safe area insets on cordova android

## Attach a .c3p
[notch size.zip](https://github.com/Scirra/Construct-3-bugs/files/5513028/notch.size.zip)

## Steps to reproduce

1. Export to Android as debug apk
2. Try with Notched phone

## Observed result

![image](https://user-images.githubusercontent.com/11213825/98591201-2db04200-22d0-11eb-8427-aaafef4a6d24.png)

PlatformInfo plugin returns 0 for all sides

## Expected result

There is a notch at the top of the screen, and it should return something.

## More details

To prove that this isn't a device specific issue, I have run a battery of tests.
- First, on web, it works perfectly: Notch is detected and Platform info returns the right values.
- Then, I tried exporting on many different Android targets, with different settings, nothing worked
- I exported as an Android Studio project and tried with the Android Emulator with a Notched phone AND with an emulated notch (in the developer settings, android can simulate about 6 or 7 types of notches). None of them worked.
- I tried getting the Notch data from android.view.window by addind code in the Fullscreen cordova plugin that would print the notch values on startup, but that didn't seem to work. I couldn't manage to write that same code in the MainActivity for some reason...

I have also tried a 3rd party addon that is supposed to get its notch information from android.view.window but the addon did not seem to work as intended, and was semi deprecated anyway.

I have to say, this is borderline infuriating to work with. Nothing works properly, and given the wide array of tests that I have done, I cannot assume that I just tested the wrong device: There is something wrong with the addon, and it doesn't work as expected. My guess is that the Cordova Webview doesn't return accurate notch info at least on android, if not on both android and IOS. After some research on the forums, it is clear that this isn't an isolated issue, and people have been getting this bug for at least 7 month.

**Affected browsers/platforms:**  Android

**First affected release:**

## System details

<details><summary>View details</summary>

PASTE HERE

</details>

**Comment by AshleyScirra** [comment]
Construct simply reports the values provided by the browser. So I don't think this will be anything to do with Construct, it'll probably be a Chrome issue.

I'm afraid at this time the UK is currently under lockdown again, and we're out the office without access to our usual range of testing devices, so it may take a long time to be able to investigate this issue further. It would probably be quicker if you have your own Android device with a notch to find a way to reproduce this independently and file your own bug with Google at crbug.com. All our code does is read the CSS value `env(safe-area-inset-top)` and the right, bottom and left variants, so if you can find a basic CSS demo using those values and reproduce with that, you have a good repro to file an issue with Google.

**Comment by AshleyScirra** [comment]
Sorry for the delay, I finally got a Nokia 7.1 which has a notch for testing with.

The original project uses a third-party addon which is not supported, so I have not used that project. I could reproduce with a quick project I made myself though: [SafeAreaInsetTest.zip](https://github.com/Scirra/Construct-3-bugs/files/6037132/SafeAreaInsetTest.zip)

It looks like an Android WebView bug which was already filed here: https://bugs.chromium.org/p/chromium/issues/detail?id=1094366
It appears Google have already written code to fix this, but it still reproduces with M88 on that device. I will follow up with them in that issue.
